### PR TITLE
Fix deserialization of Alarms and HistoricAlerts

### PR DESCRIPTION
**Pull request by mmac-m3a**
Details:
1) make all variables related to "baseline" and "alert_value" floats
2) make conversion of timestamps straight-forward avoiding unnecessary use of the `convert` function and make all of them be explicitly in the UTC timezone
3) ignore Alarm entries containing `type` attribute, because those do not seem to conform to the general schema

The last change is an ugly temporary hack. I have question regarding "non-conformant" entries pending with the subject domain expert.

### CODE REVIEW

**Review by mateuszmidor — APPROVED** [review]

File: kentik_api_library/kentik_api/requests_payload/manual_mitigations_payload.py [hunk 1]
```diff
@@ -79,7 +79,8 @@ def from_json(cls, json_string: str):
         items = list_from_json(cls.__name__, json_string)
         obj = cls()
         for i in items:
-            obj.append(from_dict(data_class=_Alarm, data=i))
+            if "type" not in i:
```
Comment (Fenthick) [inline kentik_api_library/kentik_api/requests_payload/manual_mitigations_payload.py:82]: Please add a comment that it is a workaround for "problematic" payloads

**Review by Fenthick — DISMISSED** [review]

**Comment by Fenthick** [comment]
Resolves #102

File: kentik_api_library/kentik_api/requests_payload/manual_mitigations_payload.py (hunk 1 above)
Comment (mmac-m3a) [reply kentik_api_library/kentik_api/requests_payload/manual_mitigations_payload.py:82]: Will do.

Comment (mmac-m3a) [reply kentik_api_library/kentik_api/requests_payload/manual_mitigations_payload.py]: I've added comment and at the same time made the workaround better. The `alerting.get_active_alerts` call can now return also manual mitigations :). This code needs deeper refactoring.

**Review by Fenthick — APPROVED** [review]

**Pull request merged**